assignment08/js/script.js

```diff
@@ -0,0 +1,126 @@
+window.addEventListener('load', () => {
+    
+startClock();
+    //YOUR CODE SHOULD START BELOW THIS LINE
+
+navigator.geolocation.getCurrentPosition(youAreHere);
+    
+function youAreHere(position){
+   console.log(`Latitude: `+ position.coords.latitude `, Longitude: `+ 
+   position.coords.longitude +`.`);
+}
+
+const buttons = document.getElementsByTagName('button');
+
+for(const button of buttons){
+    const dataId = button.dataset.id;
+button.addEventListener('click', () => {
+    if(dataId == 'getCurrentWX'){
+        getCurrentWX();
+    }
+    else if(dataId == 'getFiveDay') {
+            getFiveDay();
+        }
+    })
+}
+
+function getCurrentWX() {
+    console.log(`getCurrentWX() invoked`);
+    let url1 = `api.openweathermap.org/data/2.5/weather?` + `lat=` + position.coords.latitude +

```

Comment:
again you are mixing string and template syntax. This won't work.

Reply:
I couldn't test beyond this since it isn't working. I'll give you through step 4.